benchmark/singleImage.csv

```diff
@@ -0,0 +1 @@
+Debian,docker.io/library/debian:latest,"",""
```

Comment:
Do we have a list of public images we should test against?

Reply:
Great question, we originally had not included one because the only registry we had was an AWS internal preprod.  If I recall Docker hub now support referrers API.  If so we can move there, it would be great to be more public.